### PR TITLE
Upper4lower common

### PR DESCRIPTION
**Pull request by thorunna**
The most common words which are capitalized incorrectly in the development corpus were added to the config file. This includes holidays, job titles, weekdays and more. The words mentioned in writing rule 1.3.3.2 were also added, along with testing for the handling of this error class.

### CODE REVIEW

File: src/reynir_correct/config/GreynirCorrect.conf [hunk 1]
```diff
@@ -3582,6 +3582,143 @@ sjálfstæðismaður
...
+Aðfangadagskvöld
+Jóladagur
+Áramót
+Gamlársdagur
+jónsmessa
+valentínusardagur
+þorláksmessa
+
+# Job titles
+# Unable to include: félags- og barnamálaráðherra, fjármála- og efnahagsráðherra, utanríkis- og þróunarsamvinnuráðherra, utanríkis- og þróunarsamvinnuráðherra, umhverfis- og auðlindaráðherra, sjávarútvegs- og landbúnaðarráðherra, mennta- og menningarmálaráðherra, samgöngu- og sveitarstjórnarráðherra, ferðamála-, iðnaðar- og nýsköpunarráðherra, sérstakur saksóknari, sr., hr.
```
Comment (vthorsteinsson) [inline src/reynir_correct/config/GreynirCorrect.conf:3634]: These can probably be fixed by making the .conf file parser a little more intelligent.

File: src/reynir_correct/config/GreynirCorrect.conf [hunk 2]
```diff
@@ -3582,6 +3582,143 @@ sjálfstæðismaður
...
+Íþróttir
+Jarðfræði
+Kvikmyndafræði
+Landafræði
+Latína
+Líffræði
+Lýðheilsufræði
+Myndlist
+Næringarfræði
+Saga
```
Comment (vthorsteinsson) [inline src/reynir_correct/config/GreynirCorrect.conf:3701]: Be careful with this - is also a person name

File: src/reynir_correct/config/GreynirCorrect.conf [hunk 3]
```diff
@@ -3639,6 +3776,227 @@ Litháenskur
 kanada
 kanadamaður
 Kanadískur
+spánn
+Spænskur
+japan
+Japanskur
+
+# Language families
+# Unable to include: afróasískur, aímarískur, algískur, altískur, ástró-asískur, ástronesískur, dravidískur, dravídamál, eskimó-aleútískur, karíbamál, nígerkongómál, nílósaharamál, sinó-tíbetskur, suðurkákasískur, túpímál
```
Comment (vthorsteinsson) [inline src/reynir_correct/config/GreynirCorrect.conf:3785]: Missing from BÍN?

File: test/test_allkinds.py [hunk 4]
```diff
@@ -1337,4 +1405,6 @@ def test_corrected_sentences(verbose=False):
 
 
 if __name__ == "__main__":
-    pass
+    #pass
+
+    test_capitalization()
```
Comment (vthorsteinsson) [inline test/test_allkinds.py:1410]: Add newline at end of file

File: src/reynir_correct/config/GreynirCorrect.conf (hunk 2 above)
Comment (thorunna) [reply src/reynir_correct/config/GreynirCorrect.conf]: Ah of course! I replaced it with 'sagnfræði'

File: src/reynir_correct/config/GreynirCorrect.conf (hunk 3 above)
Comment (thorunna) [reply src/reynir_correct/config/GreynirCorrect.conf:3785]: Yes, they're all missing from BÍN and probably not very common anyway

File: test/test_allkinds.py (hunk 4 above)
Comment (thorunna) [reply test/test_allkinds.py]: Done!

File: src/reynir_correct/config/GreynirCorrect.conf (hunk 1 above)
Comment (vthorsteinsson) [reply src/reynir_correct/config/GreynirCorrect.conf:3641]: ...and I'm working on it.

File: test/test_allkinds.py [hunk 5]
```diff
@@ -1337,4 +1455,4 @@ def test_corrected_sentences(verbose=False):
 
 
 if __name__ == "__main__":
-    pass
+    #pass
```
Comment (vthorsteinsson) [inline test/test_allkinds.py:1458]: The pass statement is necessary here

Comment (thorunna) [reply test/test_allkinds.py]: Woops, I've fixed it!

File: src/reynir_correct/config/GreynirCorrect.conf [hunk 6]
```diff
@@ -3579,9 +3579,153 @@ Sjíti
...
+Bylting
+Uppreisn
+Meistaradeild
+Kreppa
+Endurreisn
+Jörð
+Tungl
+vesturlönd
+norðurlönd
+óskarsverðlaun 	#ekki leiðrétt, fer í 'if all(m.stofn not in stems for m in meanings)' en meanings = []
```
Comment (vthorsteinsson) [inline src/reynir_correct/config/GreynirCorrect.conf:3601]: Looks like 'óskarsverðlaun' is in BÍN, or at least in our word database. Therefore, it is not corrected. Should it definitely always be 'Óskarsverðlaun'?

Comment (vthorsteinsson) [inline src/reynir_correct/config/GreynirCorrect.conf:3601]: "óskarsverðlaun" (lowercase) seems to be in BÍN, and is therefore not corrected.

Comment (thorunna) [reply src/reynir_correct/config/GreynirCorrect.conf:3601]: According to Málfarsbankinn, it should. See here: http://malfar.arnastofnun.is/grein/67061

Comment (vthorsteinsson) [reply src/reynir_correct/config/GreynirCorrect.conf:3601]: OK, we'll make a BÍN erratum on this on our side.

**Comment by vthorsteinsson** [comment]
OK to merge this @thorunna ?

**Comment by thorunna** [comment]
Yes!

**Pull request merged**